### PR TITLE
Fix failing build

### PR DESCRIPTION
**Pull request by kristofferlund**
##### Description of Change

This fix the build that is failing at master branch. It also cleans up some tests that were written wrong, and by request from Anders changes the warning message that is displayed when using negative values with logarithmic scales. 
The csv for line chart has been changed to not use a "wrong" data set as a default example. 
##### Checklist

- [x] PR description included and it is linked to an issue
- [x] `make test` and `make lint` passes
- [x] tests are changed or added
- [x] relevant documentation is changed or added

Notes:

### CODE REVIEW

File: visualizations/bar_chart/webviz_bar_chart/__init__.py [hunk 1]
```diff
@@ -14,22 +14,33 @@ class BarChart(FilteredPlotly):
...
+            *args, **kwargs):
         self.logy = logy
         super(BarChart, self).__init__(
                 data,
                 *args,
                 layout={
                     'barmode': barmode,
-                    'xaxis': {'title': xaxis},
-                    'yaxis': {'title': yaxis, 'type': 'log' if logy else '-'}
+                    'xaxis': {'title': xtitle, 'range': xrange},
```
Comment (anders-kiaer) [inline visualizations/bar_chart/webviz_bar_chart/__init__.py]: Change variable name from Python built-in `xrange` to something else?

File: visualizations/bar_chart/webviz_bar_chart/__init__.py [hunk 2]
```diff
@@ -14,22 +14,33 @@ class BarChart(FilteredPlotly):
         ``'overlay'``. Defines how multiple bars per index-value are combined.
         See `plotly.js layout-barmode <https://plot.ly/javascript/reference/
         #layout-barmode>`_.
-    :param xaxis: Will create a label for the x-axis. Defaults to `None`.
-    :param yaxis: Will create a label for the y-axis. Defaults to `None`.
+    :param xrange: list of minimum and maximum value. Ex: [3, 15].
+    :param yrange: list of minimum and maximum value. Ex: [3, 15].
+    :param xtitle: Will create a label for the x-axis. Defaults to `None`.
+    :param ytitle: Will create a label for the y-axis. Defaults to `None`.
```
Comment (anders-kiaer) [inline visualizations/bar_chart/webviz_bar_chart/__init__.py]: The other option here is to use `xlabel` and `ylabel` (which users coming from matplotlib are more used to). But at the same time `xtitle` and `ytitle` is close to plotly API. No strong opinion here, as long as it has been considered :)

File: visualizations/fan_chart/webviz_fan_chart/__init__.py [hunk 3]
```diff
@@ -170,21 +172,28 @@ def __init__(
             self,
             data,
             observations=None,
-            logx=False,
-            logy=False,
+            logx=False, logy=False,
+            xrange=None, yrange=None,
```
Comment (anders-kiaer) [inline visualizations/fan_chart/webviz_fan_chart/__init__.py]: According to [plotly documentation on `range`](https://plot.ly/javascript/reference/#layout-yaxis-range), if the axis is logarithmic, the values in `range` should be the values after taking `log`.

I think we should do that `log`-transformation for the user. I.e. add the transformation of `[x,y]range` within the corresponding `PageElement` (given that `log[x,y] = True`)

File: visualizations/line_chart/tests/test_line_chart.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import unittest
+import pandas as pd
+import warnings
+from webviz_line_chart import LineChart
+
+line_mock_data = {
```
Comment (kristofferlund) [inline visualizations/line_chart/tests/test_line_chart.py]: Just nice to have if we want to write more tests for this component in the future.

**Pull request merged**